src/types/decimal.rs

```diff
@@ -161,10 +185,7 @@ impl From<BigDecimal> for Decimal {
         let (big_int_coefficient, negative_exponent) = value.as_bigint_and_exponent();
         // Discard the BigInt coefficient's sign before converting it to a BigUint to ensure
         // the conversion succeeds.
-        let magnitude: BigUint = big_int_coefficient
-            .abs()
-            .to_biguint()
-            .expect("abs() would have prevented a negative number");
```

Comment:
`cargo fmt` seems to have replaced this `expect` with an `unwrap`. I'm unsure why that's the preference, but I'm ok with it. 🤷